bcda/bcdacli/cli.go

```diff
@@ -362,6 +363,17 @@ func setUpApp() *cli.App {
 			Action: func(c *cli.Context) error {
 				ignoreSignals()
 				success, failure, skipped, err := cclf.ImportCCLFDirectory(filePath)
+				if err != nil {
+					log.API.Error("error returned from ImportCCLFDirectory: ", err)
+					cli.NewExitError("an error occurred during the cclf-import process, see logs for more details", 1)
+					return err
+				}
+				if failure > 0 || skipped > 0 {
+					log.API.Error("Successfully imported %v files.  Failed to import %v files.  Skipped %v files.  See logs for more details. ", err)
+					cli.NewExitError("some files failed to import or were skipped, see logs for more details", 1)
+					return err
+				}
+				log.API.Infof("Completed CCLF import.  Successfully imported %v files.  Failed to import %v files.  Skipped %v files.  See logs for more details.", success, failure, skipped)
```

Comment:
@laurenkrugen-navapbc last question, if the instances where "failure" and "skipped" are populated are handled in the condition above, and there's a return statement in said condition -- would that not exit out of the function before getting to this log?

Reply:
If there is an error or a file is skipped, our import function will increment [those counters](https://github.com/CMSgov/bcda-app/blob/master/bcda/cclf/cclf.go#L332C15-L332C26) as opposed to returning an error. The error returned doesn't really go anywhere right now; the jenkins pipeline script doesn't check the output of the import job so those won't impact the outcome of the pipeline run.

Reply:
@laurenkrugen-navapbc would you be open to discuss this further during eng office hours? I'm not fully clear re: those counters and am curious why the jenkins pipeline doesn't check the output of the import job.